docs/user-guide/certificate-configuration-scenarios.md

```diff
@@ -290,6 +289,13 @@ this field is not defined, the `zwe init` command uses the value `zowe.externalD
      - "zOSMFCA"
 ```
 
+If you would like to use this example in your Zowe configuration YAML file, you just need to replace four fields with your own ones.
+
+1. Replace `ZoweKeyringZOSMF` with the your own key ring name.
+2. Replace `IZUSVR` with your user name.
+3. Replace `DefaultzOSMFCert.IZUDFLT` with your own label name.
+4. Replace `zOSMFCA` with your certificate authority name.
```

Comment:
4. Replace `zOSMFCA` with the certificate authority that has been used to sign the certificate used in step 3.